{
  "cveId": "CVE-2024-52900",
  "eventName": "Initial Analysis",
  "cveChangeId": "521BEAE3-E409-4D10-8B4C-082749BBBAB9",
  "sourceIdentifier": "nvd@nist.gov",
  "created": "2025-07-01T18:07:20.727",
  "details": [
    {
      "action": "Added",
      "type": "CVSS V3.1",
      "newValue": "AV:N/AC:L/PR:L/UI:R/S:C/C:L/I:L/A:N"
    },
    {
      "action": "Added",
      "type": "CPE Configuration",
      "newValue": "OR\n          *cpe:2.3:a:ibm:cognos_analytics:11.2.4:fixpack1:*:*:*:*:*:*\n          *cpe:2.3:a:ibm:cognos_analytics:11.2.4:-:*:*:*:*:*:*\n          *cpe:2.3:a:ibm:cognos_analytics:*:*:*:*:*:*:*:* versions from (including) 11.2.0 up to (excluding) 11.2.4\n          *cpe:2.3:a:ibm:cognos_analytics:11.2.4:fixpack2:*:*:*:*:*:*\n          *cpe:2.3:a:ibm:cognos_analytics:11.2.4:fixpack3:*:*:*:*:*:*\n          *cpe:2.3:a:ibm:cognos_analytics:*:*:*:*:*:*:*:* versions from (including) 12.0.0 up to (excluding) 12.0.4\n          *cpe:2.3:a:ibm:cognos_analytics:11.2.4:fixpack4:*:*:*:*:*:*\n          *cpe:2.3:a:ibm:cognos_analytics:11.2.4:fixpack5:*:*:*:*:*:*\n          *cpe:2.3:a:ibm:cognos_analytics:12.0.4:-:*:*:*:*:*:*\n          *cpe:2.3:a:ibm:cognos_analytics:11.2.4:interim_fix_1:*:*:*:*:*:*\n          *cpe:2.3:a:ibm:cognos_analytics:11.2.4:interim_fix_2:*:*:*:*:*:*\n          *cpe:2.3:a:ibm:cognos_analytics:11.2.4:interim_fix_3:*:*:*:*:*:*\n          *cpe:2.3:a:ibm:cognos_analytics:11.2.4:interim_fix_4:*:*:*:*:*:*\n          *cpe:2.3:a:ibm:cognos_analytics:11.2.4:interim_fix_5:*:*:*:*:*:*"
    },
    {
      "action": "Added",
      "type": "Reference Type",
      "newValue": "IBM Corporation: https://www.ibm.com/support/pages/node/7238163 Types: Patch, Vendor Advisory"
    }
  ]
}